DataFormats/L1TrackTrigger/interface/TTTrack_TrackWord.h

```diff
@@ -231,6 +231,18 @@ class TTTrack_TrackWord {
   double getMVAQuality() const { return tqMVABins[getMVAQualityBits()]; }
   double getMVAOther() const { return getMVAOtherBits(); }
 
+  // Get pre-sigmoid TQ MVA bins
+  static constexpr std::array<double, 1 << TrackBitWidths::kMVAQualitySize> getTqMVAPreSigBins() {
+    return {{-480,
```

Comment:
This should be -16 at least for the ap_fixed<10,5> case, though is entirely dependent on what digitisation you have chosen for the BDT so is a hard coded solution. Another reason for this not to be in the track word and instead derived from the precision of the BDT in the KF out.